sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -309,29 +289,35 @@ def _get_next_cb(self, continuation_token, **kwargs):  # pylint: disable=inconsi
 
     def _extract_data_cb(self, get_next_return):
         self._location_mode, self._response, self._headers = get_next_return
-        props_list = [
-            TableItem._from_generated(t, **self._headers)  # pylint: disable=protected-access
-            for t in self._response.value
-        ]
+        props_list = [TableItem._from_generated(t) for t in self._response.value]  # pylint: disable=protected-access
         return self._headers[NEXT_TABLE_NAME] or None, props_list
 
 
 class TableEntityPropertiesPaged(PageIterator):
-    """An iterable of TableEntity properties.
-
-    :param callable command: Function to retrieve the next page of items.
-    :param str table: The name of the table.
-    :keyword int results_per_page: The maximum number of results retrieved per API call.
-    :keyword str filter: The filter to apply to results.
-    :keyword str select: The select filter to apply to results.
-    :keyword str continuation_token: An opaque continuation token.
-    """
-
-    def __init__(self, command, table, **kwargs):
+    """An iterable of TableEntity properties."""
+
+    table: str
+    """The name of the table."""
+    results_per_page: Optional[int]
+    """The maximum number of results retrieved per API call."""
+    filter: Optional[str]
+    """The filter to apply to results."""
+    select: Optional[str]
+    """The select filter to apply to results."""
+
+    def __init__(self, command: Callable, table: str, **kwargs: Any) -> None:
+        """
+        :param callable command: Function to retrieve the next page of items.
+        :param str table: The name of the table.
+        :keyword int results_per_page: The maximum number of results retrieved per API call.
+        :keyword str filter: The filter to apply to results.
+        :keyword str select: The select filter to apply to results.
+        :keyword Optional[str] continuation_token: An opaque continuation token.
+        """
         super(TableEntityPropertiesPaged, self).__init__(
             self._get_next_cb,
             self._extract_data_cb,
-            continuation_token=kwargs.get("continuation_token") or {},
+            continuation_token=kwargs.get("continuation_token"),
```

Comment:
Why was the default removed here?

Reply:
I don't understand why we defaulted to an empty dict `{}`. Same question in `TablePropertiesPaged`, defaulted to an empty string `""`. Actually, this parameter is `Optional` in parent class, it's allowed to pass it in value `None` unless empty string has a special meaning. But `{}` is type incompatible, we should update the default value here anyway.
https://github.com/Azure/azure-sdk-for-python/blob/81dbf845756f5fbb2e54f96394100f408ec80512/sdk/core/azure-core/azure/core/paging.py#L47-L53